### ISSUE TITLE
transcoder, flac to mp3

### ISSUE THREAD

**Issue by Clementine-Issue-Importer** [clementine-player/Clementine#3640]
_From [xxttrree...@gmail.com](https://code.google.com/u/105205058448633795635/) on April 28, 2013 17:30:58_

arch x64 with all gstreamer plugins installed 

Sun Apr 28 16:23:58 2013: Transcoding 1 files using 2 threads
Sun Apr 28 16:23:58 2013: Starting /run/media/kremix/sTorE/hipster/santogold/santogold/09-unstoppable.flac
Sun Apr 28 16:23:58 2013: Using 'lamemp3enc' (rank 256)
Sun Apr 28 16:23:58 2013: Adding xingmux and id3v2mux to the pipeline
Sun Apr 28 16:23:58 2013: Setting lamemp3enc property: target = 1
Sun Apr 28 16:23:58 2013: Setting lamemp3enc property: bitrate = 320
Sun Apr 28 16:23:58 2013: Setting lamemp3enc property: cbr = true
Sun Apr 28 16:23:58 2013: Setting lamemp3enc property: quality = 0
Sun Apr 28 16:23:58 2013: Setting lamemp3enc property: encoding-engine-quality = 2
Sun Apr 28 16:23:58 2013: Setting lamemp3enc property: mono = false
Sun Apr 28 16:23:58 2013: Could not create the GStreamer element "id3v2mux" - make sure you have all the required GStreamer plugins installed
Sun Apr 28 16:23:58 2013: Couldn't find an encoder for audio/mpeg, mpegversion=(int)1, layer=(int)3, check you have the correct GStreamer plugins installed

_Original issue: http://code.google.com/p/clementine-player/issues/detail?id=3640_

**Comment by paperbagcorner** [comment]
I don't think this is an issue anymore. The latest git version works for me on arch x64, where id3v2mux is in the package `gst-plugins-good`.
